build_containers/author-build.dockerfile

```diff
@@ -1,13 +0,0 @@
-FROM ubuntu:16.04
```

Comment:
This is used in other builds so we shouldn't remove this 
https://github.com/ONSdigital/eq-deploy/pull/45/files#diff-c5a2c990f03bcdb3e6d4e81cf49d64f5R142

Reply:
The only places I could see it used was in the smoke tests and the author build. Is there somewhere else?

Reply:
Removed from all locations.